Failed: HubSpec

This has failed before but it was some time ago so raising a new issue:

https://github.com/akka/akka/issues/21391

```
[info] - must properly signal error to consumers *** FAILED *** (2 seconds, 20 milliseconds)
[info]   java.lang.AssertionError: assertion failed: expected OnNext(1), found OnNext(3)
[info]   at scala.Predef$.assert(Predef.scala:219)
[info]   at akka.testkit.TestKitBase.expectMsg_internal(TestKit.scala:402)
[info]   at akka.testkit.TestKitBase.expectMsg(TestKit.scala:378)
[info]   at akka.testkit.TestKitBase.expectMsg$(TestKit.scala:378)
[info]   at akka.testkit.TestKit.expectMsg(TestKit.scala:882)
[info]   at akka.stream.testkit.TestSubscriber$ManualProbe.$anonfun$expectNextN$1(StreamTestKit.scala:387)
[info]   at akka.stream.testkit.TestSubscriber$ManualProbe$$Lambda$823/1297969700.apply(Unknown Source)
[info]   at scala.collection.Iterator.foreach(Iterator.scala:929)
```

--- a/akka-stream-tests/src/test/scala/akka/stream/scaladsl/HubSpec.scala
+++ b/akka-stream-tests/src/test/scala/akka/stream/scaladsl/HubSpec.scala
@@ -313,6 +313,10 @@ class HubSpec extends StreamSpec {
       downstream1.request(4)
       downstream2.request(8)
 
+      // sending the first element is in a race with downstream subscribing
+      // give a bit of time for the downstream to complete subscriptions
+      Thread.sleep(50)
+
       (1 to 8) foreach (upstream.sendNext(_))
 
       downstream1.expectNext(1, 2, 3, 4)
@@ -575,6 +579,10 @@ class HubSpec extends StreamSpec {
 
       downstream1.request(4)
       downstream2.request(8)
+
+      // to make sure downstream subscriptions are done before
+      // starting to send elements
+      Thread.sleep(50)
 
       (0 until 16) foreach (upstream.sendNext(_))
 

